packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerBestPractices.md

```diff
@@ -8,11 +8,13 @@
 - Maximum recommended number of swatches per row is 8.
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31360&commitId=79c05e363bdc5f40c267234ce476a9a849a0b222&env=prod&clientType=FLUENTUI)


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>